python-checks/src/test/java/org/sonar/python/checks/hotspots/GraphQLIntrospectionCheckTest.java

```diff
@@ -19,15 +19,16 @@
  */
 package org.sonar.python.checks.hotspots;
 
-import java.util.Collections;
+import java.util.List;
+
 import org.junit.jupiter.api.Test;
 import org.sonar.python.checks.utils.PythonCheckVerifier;
 
 class GraphQLIntrospectionCheckTest {
 
   @Test
   void test() {
-    PythonCheckVerifier.verify(Collections.singletonList("src/test/resources/checks/hotspots/graphQLIntrospection.py"), new GraphQLIntrospectionCheck());
+    PythonCheckVerifier.verify(List.of("src/test/resources/checks/hotspots/graphQLIntrospection.py"), new GraphQLIntrospectionCheck());
```

Comment:
In fact I don't think you even need the list at all. This API works both with a single path an with a list of paths.
With a list, it will create a project symbol table for all the files, but this is unneeded for a single file.